pkg/operator/ceph/cluster/status/conditions.go

```diff
@@ -0,0 +1,300 @@
+/*
```

Comment:
Please add some unit tests for this file

Reply:
Sure